parlai/core/dict.py

```diff
@@ -690,6 +713,9 @@ def save(self, filename=None, append=False, sort=True):
         # save opt file
         with open(filename + '.opt', 'w', encoding='utf-8') as handle:
             json.dump(self.opt, handle, indent=4)
+        # save the byte level bpe model file as well
+        if self.tokenizer == 'bytelevelbpe':
+            self.byte_level_bpe.tokenizer(os.path.dirname(filename), filename)
```

Comment:
is this right? This doesn't look right. Is the suffix implied? Is `tokenizer()` really the API call for the save? This just looks strange...

Reply:
Thanks for flagging this, I will double check on this.